source/ceylon/test/engine/internal/file.ceylon

```diff
@@ -0,0 +1,39 @@
+import java.io {
+    File,
+    FileOutputStream,
+    PrintWriter
+}
+
+native class FileFacade(String path) satisfies Destroyable {
```

Comment:
no nothing particularly (maybe at that time it did't work, but I don't remember) ... feel free to use your version in HtmlReporter also